### ISSUE TITLE
TypeError: Cannot read property 'status' of undefined

### ISSUE THREAD

**Issue by talisk** [Microsoft/code-push#295]
When I run `code-push deployment ls AppName`, I find an error.

``` bash
/Users/username/.nvm/versions/node/v5.12.0/lib/node_modules/code-push-cli/node_modules/code-push/script/management-sdk.js:356
                        statusCode: res.status
                                       ^

TypeError: Cannot read property 'status' of undefined
    at /Users/username/.nvm/versions/node/v5.12.0/lib/node_modules/code-push-cli/node_modules/code-push/script/management-sdk.js:356:40
    at Request.callback (/Users/username/.nvm/versions/node/v5.12.0/lib/node_modules/code-push-cli/node_modules/superagent/lib/node/index.js:687:12)
    at ClientRequest.<anonymous> (/Users/username/.nvm/versions/node/v5.12.0/lib/node_modules/code-push-cli/node_modules/superagent/lib/node/index.js:639:10)
    at emitOne (events.js:90:13)
    at ClientRequest.emit (events.js:182:7)
    at TLSSocket.socketErrorListener (_http_client.js:306:9)
    at emitOne (events.js:90:13)
    at TLSSocket.emit (events.js:182:7)
    at emitErrorNT (net.js:1265:8)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)
```

I'm in China, so my network is unstable. But I think the exception handler has some issue.

**Comment by cloudkay** [comment]
i got the same issue

**Comment by silhouettes** [comment]
Thanks for reporting this! This has been fixed in v1.12.5-beta: https://github.com/Microsoft/code-push/releases/tag/v1.12.5-beta
